src/test/java/org/prebid/server/bidder/colossus/ColossusBidderTest.java

```diff
@@ -51,9 +53,8 @@ public void creationShouldFailOnInvalidEndpointUrl() {
     public void makeHttpRequestsShouldReturnErrorIfImpExtCouldNotBeParsed() {
         // given
         final BidRequest bidRequest = BidRequest.builder()
-                .imp(singletonList(Imp.builder()
-                        .ext(mapper.valueToTree(ExtPrebid.of(null, mapper.createArrayNode())))
-                        .build()))
+                .imp(singletonList(givenImp(impBuilder -> impBuilder
```

Comment:
final BidRequest bidRequest = givenBidRequest(impBuilder -> impBuilder
                        .ext(mapper.valueToTree(ExtPrebid.of(null, mapper.createArrayNode()))), identity());